app/views/admin/admin_users/index.html.erb

```diff
@@ -14,7 +14,13 @@
       <td><%= mail_to user.email %></td>
       <td><%= user.role %></td>
       <td><%= user.account_disabled ? 'Yes' : '' %></td>
-      <td class="action last"><%= cms_delete_link user, :url => admin_admin_user_path(user) %></td>
+      <td class="action last">
+        <% if user != current_user %>
+          <%= cms_delete_link user, :url => admin_admin_user_path(user) %>
+        <% else %>
+          &nbsp;
```

Comment:
Just hides the delete button on the user row that represents the current user - not really front end dev. What would you suggest instead?